packages/p2p/src/components/my-ads/edit-ad-form.jsx

```diff
@@ -65,11 +65,13 @@ const EditAdForm = () => {
         my_ads_store.setEditAdFormError('');
 
         if (payment_method_names && !payment_method_details) {
+            const selected = [];
```

Comment:
Please be as descriptive as you can be so the code is easily readable. 

```suggestion
            const selected_methods = [];
```

Reply:
`selected_methods` is already a local state in line 44 😅, but I'll change this variable into a more readable name